galaxy_ng/tests/integration/api/test_artifact_upload.py

```diff
@@ -57,7 +57,7 @@ def test_api_publish(ansible_config, artifact, upload_artifact, use_distribution
     which should indicate a successful import.
     """
 
-    config = ansible_config("basic_user")
+    config = ansible_config("partner_engineer")
```

Comment:
I think we can use `basic_user` if we set it up  in stage env like here: https://github.com/ansible/galaxy_ng/blob/28ad4b525abdeb80a0c88a60b2dea9a205352c02/dev/common/setup_test_data.py#L88-L96

I see `basic_user` is tied to `aa_no_access` ... as long as that user has ansible entitlements, we can add their rh-identity group as one of the namespace owners for `autohubtest2` (which currently has 3 groups as namespace owners).

I don't see an `autohubtest3` namespace in stage, and I only see it used in one standalone-only test, so we can probably leave that as-is.

Reply:
We should fix the tests so that these namespaces aren't assumed to pre-exist. The original IQE tests created them on the fly as part of client spinup or something like that, but we can now be more direct and simply create a randomized namespace for each test as needed instead of relying on static names.